libs/vscode/migrate/src/lib/commands/migrate-commands.ts

```diff
@@ -93,10 +106,9 @@ export async function confirmPackageChanges() {
             );
           });
 
-        repository.add(filesToAdd);
+        await repository.add(filesToAdd);
 
-        repository.commit('chore: start nx migration', {
-          signCommit: true,
+        await repository.commit('chore: start nx migration', {
```

Comment:
`signCommit` was not working unless you have it set up in your git config. Safer to not use it.

Also had to add `await` here, otherwise the initial package.json changes were never committed, and it gets lumped into the same commit as the first migration.